templates/base.html

```diff
@@ -1,83 +1,135 @@
 <!DOCTYPE html>
 
 <html>
-<head>
-  <title>{% block title%}Web & Design portal{% endblock %}</title>
-  <link rel="stylesheet" type="text/css" media="screen" href="{{ versioned_static('css/styles.css') }}">
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <link rel="shortcut icon" href="https://assets.ubuntu.com/v1/0843d517-favicon.ico" type="image/x-icon" />
-</head>
+  <head>
+    <title>{% block title%}Web & Design portal{% endblock %}</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      media="screen"
+      href="{{ versioned_static('css/styles.css') }}"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="shortcut icon"
+      href="https://assets.ubuntu.com/v1/0843d517-favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
 
-<body>
-  <main class="p-content">
-
-    <header id="navigation" class="p-navigation">
-      <div class="p-navigation__row">
-        <div class="p-navigation__banner">
-          <div class="p-navigation__tagged-logo">
-            <a class="p-navigation__link" href="/">
-              <div class="p-navigation__logo-tag">
-                <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
-              </div>
-              <span class="p-navigation__logo-title">Canonical</span>
-            </a>
+  <body>
+    <main class="p-content">
+      <header id="navigation" class="p-navigation">
+        <div class="p-navigation__row">
+          <div class="p-navigation__banner">
+            <div class="p-navigation__tagged-logo">
+              <a class="p-navigation__link" href="/">
+                <div class="p-navigation__logo-tag">
+                  <img
+                    class="p-navigation__logo-icon"
+                    src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg"
+                    alt=""
+                  />
+                </div>
+                <span class="p-navigation__logo-title">Canonical</span>
+              </a>
+            </div>
+            <a
+              href="#navigation"
+              class="p-navigation__toggle--open"
+              title="menu"
+              >Menu</a
```

Comment:
I think Prettier doesn't format HTML very well, it tends to break it html closing tags incorrectly.

Reply:
Thanks! I reverted the formatting.